vendor/bat-native-ads/src/bat/ads/internal/eligible_ads/round_robin_advertisers.h

```diff
@@ -22,10 +21,8 @@ T FilterSeenAdvertisers(const T& ads,
   const auto iter = std::remove_if(
       unseen_advertisers.begin(), unseen_advertisers.end(),
       [&seen_advertisers](const CreativeAdInfo& creative_ad) {
-        SplitTestFrequencyCap frequency_cap;
-        return frequency_cap.ShouldExclude(creative_ad) ||
-               seen_advertisers.find(creative_ad.advertiser_id) !=
-                   seen_advertisers.end();
```

Comment:
Frequency capping now occurs before filtering creative ads for seen ads/advertisers, so this fix is no longer required.